docs/integrations/spark.md

```diff
@@ -11,6 +11,16 @@ redirect_from: ../using/spark.html
 # Using lakeFS with Spark
 {: .no_toc }
 [Apache Spark](https://spark.apache.org/) is a unified analytics engine for big data processing, with built-in modules for streaming, SQL, machine learning and graph processing.
+
+lakeFS support in Spark has two tiers:
+
+* Access lakeFS using the S3A gateway.
+* Access lakeFS using the lakeFS-specific Hadoop FileSystem.
+
+Using the S3A gateway is easier to configure and may be more suitable for legacy or
+small-scale applications.  Using the lakeFS FileSystem requires somewhat more complex
```

Comment:
This is really helpful..! what do you think of adding the why to the beginning of the explanation of each tier? or alternatively linking the word "S3A gateway" to line 40 and lakeFS FileSystem to line 183

Reply:
Good idea -- added links!